gulp/tasks/styleguide.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var gulp       = require('gulp');
+var styleguide = require('sc5-styleguide');
+var config     = require('../config');
+
+gulp.task('styleguide:generate', function() {
+  return gulp.src(config.styles.src)
+    .pipe(styleguide.generate({
+        title: 'The Shelf Styleguide',
+        server: true,
+        port: 4000,
+        rootPath: config.styleguide.dest,
+        overviewPath: 'README.md'
+      }))
+    .pipe(gulp.dest(config.styleguide.dest));
+});
+
+gulp.task('styleguide:applystyles', function() {
+  return gulp.src(config.styles.dest + '/main.css')
+    .pipe(styleguide.applyStyles())
+    .pipe(gulp.dest(config.styleguide.dest));
+});
+
+gulp.task('watch-styleguide', ['styleguide'], function() {
```

Comment:
*sc5-styleguide* has its own server and runs by itself.
So running ```gulp watch-styleguide``` launches the styleguide's server in port *4000*. It's ok for now, but in the future I'd like the styleguide to live withing the actual app.
I'll add this new task to the Wiki once this branch is merged